client/support/support-user/login-dialog.jsx

```diff
@@ -31,6 +31,26 @@ const SupportUserLoginDialog = React.createClass( {
 		this.setState( { supportPassword: '' } );
 	},
 
+	onInputKeyDown( event ) {
+		switch ( event.key ) {
+			case 'Enter':
+				event.preventDefault();
+				switch ( event.target.name ) {
+					case 'supportUser':
+						console.log( this.supportPasswordInput );
+						this.supportPasswordInput.focus();
+						break;
+					case 'supportPassword':
+						this.onChangeUser();
+						break;
```

Comment:
Just observing that here we're using `this` and below, for a similar function call, we're using `this.props`. I guess maybe that's because the function below, which operates specifically on this dialog, is passed in as a property, but here the `onChangeUser()` function is at sort of a higher level and so isn't passed as a property to the dialog. Should we pass `onChangeUser()` in as a prop or is this sort of usage acceptable? You've read and written more Calypso code than I have, so I trust your instinct and just wanted to make sure this wasn't just an oversight, if a prop is called for.